Author: Agustin Martin Domingo <agmartin@debian.org>
Description: Handle perl 5.22 deprecation of unescaped left brace in regular expressions.
Bug-Debian: https://bugs.debian.org/826507

This is to deal with new perl 5.22 warning about that

Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\\indexentry{ <-- HERE (.+)--([8-9a-f][0-9a-f])\}\{1\}/ at ./make-inp-rules.pl line 11.

--- a/make-rules/inputenc/make-inp-rules.pl
+++ b/make-rules/inputenc/make-inp-rules.pl
@@ -8,7 +8,7 @@
 
 
 while (<STDIN>) {
-  if (/\\indexentry{(.+)--([8-9a-f][0-9a-f])\}\{1\}/) {
+  if (/\\indexentry\{(.+)--([8-9a-f][0-9a-f])\}\{1\}/) {
     if (!($1 =~ /inputenc Error|\@inpenc\@undefined/)) {
       $i = hex($2);
       $macro = $1;
